src/monitor/Monitor.ts

```diff
@@ -255,6 +258,12 @@ export class Monitor {
         notificationPath: "across-invalid-fills",
       });
     });
+
+    this.logger.info({
+      at: "Monitor::invalidFillsByChain",
+      message: "Invalid fills by chain",
+      invalidFillsByChainId,
```

Comment:
I think the notification path needs to change, since it's good to keep spam away from the monitor channel as there are so many potentially important things logged there.
```suggestion
      invalidFillsByChainId,
      notificationPath: "across-invalid-fills",
```

Reply:
[28cb41b](https://github.com/across-protocol/relayer/pull/2589/commits/28cb41bff5606ef86da2368cffcc0e12d0788794)